netlogo-gui/src/main/app/FileManager.scala

```diff
@@ -314,6 +314,8 @@ class FileManager(workspace: AbstractWorkspaceScala,
           case e: Exception => println("Exception in FileMenu.loadModel: " + e)
           None
         })
+    new LoadErrorEvent().raise(eventRaiser)
```

Comment:
This appears to raise the load error on every external change.  Shouldn't it only get raised on an actual error?  I also noticed that with these changes, when NetLogo reloads on an external changes it switches to the interface tab on a successful recompile.  I suspect that's related to this.  I at least confirmed the behavior on current `hexy` is to stay on the open code tab.

Reply:
You're right! Somehow I completely missed that.